docs/content/Contribution guidelines/operations/release-management.md

```diff
@@ -30,9 +30,9 @@ change the version in the following line:
 syncer_image="quay.io/kubestellar/syncer:{{ config.ks_next_tag }}"
 ```
 
-### Update the core-helm-chart 'Chart.yaml' and 'values.yaml' files with a reference to the new version of the KubeStellar Helm chart version
+### Update the chart 'Chart.yaml' and 'values.yaml' files with a reference to the new version of the KubeStellar Helm chart version
```

Comment:
This document is obsolete. Please do not muddy the waters by trying to change it partially. The new release procedure is being developed in Issue #1732 and PR #1758 .

Reply:
change undone